src/pivotal-ui/components/panels/activate-upgrade-panel.js

```diff
@@ -0,0 +1,14 @@
+var upgradePanel;
+var handleInputChangeForUpgrade = function handleInputChangeForUpgrade(e){
+  if(e.detail.currentValue === upgradePanel.getAttribute('data-username')){
+    //jQuery because using bootstrap collapse functionality
+    $(upgradePanel).collapse('show');
+    document.removeEventListener('FormInput:input', handleInputChangeForUpgrade);
+  }
+};
+
+document.addEventListener('DOMContentLoaded', function(){
```

Comment:
this was originally written in vanilla js so it could be reused if we ever aren't using jquery (see: react). But then we got stuck with jQ because bootstrap functionality.

I could wrap the entire thing in the `DOMContentLoaded` callback, but I like to define my functions before that point.